makefile

```diff
@@ -79,6 +79,28 @@ package-rpm:
 	$(BGO_SPACE)/Tool/src/packaging/linux/build_rpm_linux.sh amd64
 	$(BGO_SPACE)/Tool/src/packaging/linux/build_rpm_linux.sh arm64
 
+# This will be removed in the next major version release
+build-package-legacy:
+	@echo "===Building legacy artifacts with older names==="
+	cd $(BGO_SPACE)
+
+	@echo "Building and packaging legacy artifacts for Linux"
+	cp $(BGO_SPACE)/build/dist/aws-xray-daemon-linux-amd64-${VERSION}.zip $(BGO_SPACE)/build/dist/aws-xray-daemon-linux-${VERSION}.zip
```

Comment:
I don't think we're supposed to be using `${VERSION}` for the legacy ones. As we won't publish them for 4.x, a hardcoded `3.x` should be fine

Reply:
I agree. Fixed version name as `3.x` for legacy artifacts